contracts/marketplace/src/execute.rs

```diff
@@ -35,6 +35,8 @@ pub fn instantiate(
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
+    // FIXME: add validations for ask + bid expiry
```

Comment:
@jhernandezb There's nothing to validate here since the expiry range hasn't been set yet. What were you thinking to validate here?

Reply:
```
msg.ask_expiry.min < msg.ask_expiry.max
msg.bid_expiry.min < msg.bid_expiry.max
```